emcc.py

```diff
@@ -149,6 +149,7 @@
 VALID_ENVIRONMENTS = ('web', 'webview', 'worker', 'node', 'shell')
 SIMD_INTEL_FEATURE_TOWER = ['-msse', '-msse2', '-msse3', '-mssse3', '-msse4.1', '-msse4.2', '-mavx']
 SIMD_NEON_FLAGS = ['-mfpu=neon']
+LINK_ONLY_FLAGS = set(['--embed-file', '--exclude-file', '--preload-file'])
```

Comment:
How about `--pre-js`, `--minify`, `--closure`, `--post-js`, `--js-transform`, `--closure-args`, `--extern-post-js`, `--extern-pre-js`, `--oformat`?

Reply:
And `--profiling`, `--memoryprofiler`, `--tracing`. `--emit-symbol-map` `--use-preload-plugins`, `--ignore-dynamic-linking`, `--js-library`, `--memory-init-file`, `--proxy-to-worker`

Reply:
Added those in.

Reply:
For `--minify`, it says "[same as -g1 if passed at compile time, otherwise applies at link]", should we warn on it?

Reply:
Oh ok .. leave that one out then.